website/views/index.jade

```diff
@@ -13,34 +13,43 @@ html(ng-app="habitrpg", ng-controller="RootCtrl", ng-class='{"applying-action":a
     meta(name='apple-mobile-web-app-capable', content='yes')
     meta(name='mobile-web-app-capable', content='yes')
 
-    include ./shared/new-relic
-    //FIXME for some reason this won't load when in footerCtrl.js#deferredScripts()
-    script(type="text/javascript", src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-5016f6cc44ad68a4", async="async")
+    != env.getManifestFiles("app", "css")
 
-    script(type='text/javascript').
-     window.env = !{JSON.stringify(env._.pick(env, env.clientVars))};
-
-    != env.getManifestFiles("app")
+    // Does not work correctly inside .css file
+    style(type='text/css').
+      [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
+        display: none !important;
+      }
 
     //webfonts
     link(href='//fonts.googleapis.com/css?family=Lato:300,400,700,400italic,700italic', rel='stylesheet', type='text/css')
+  body
+    div#loadingScreen(ng-if="!appLoaded")
```

Comment:
The div is unnecessary when there's an id or a class, so this can be:

```jade
#loadingScreen(ng-if="!appLoaded")
  h1 Loading
.ng-cloak(ng-if="appLoaded", ng-controller='GroupsCtrl')
```